gitlab-runner-17.1.yaml

```diff
@@ -27,15 +30,22 @@ pipeline:
     with:
       deps: github.com/Azure/azure-sdk-for-go/sdk/azidentity@v1.6.0 github.com/hashicorp/go-retryablehttp@v0.7.7
 
+  - runs: |
+      mkdir -p "${{targets.contextdir}}"/
+      cp dockerfiles/runner/alpine/entrypoint "${{targets.contextdir}}"/entrypoint
+
   - uses: go/build
     with:
       packages: .
       output: gitlab-runner
       ldflags: -s -w -X gitlab.com/gitlab-org/gitlab-runner/common.NAME=${{package.name}} -X gitlab.com/gitlab-org/gitlab-runner/common.VERSION=v${{package.version}}
 
 subpackages:
-  - name: gitlab-runner-helper
+  - name: gitlab-runner-helper-17.1
```

Comment:
```
var-transforms:
  - from: ${{package.version}}
    match: ^(\d+\.\d+)\.\d+$
    replace: "$1"
    to: short-package-version
```

If you add this block after the `package` definition, then the subpackage name can be something like `gitlab-runner-helper-${{vars.short-package-version}}`

How's that seem to you @developer-guy?

Reply:
Try again now @Pyronewbic with the change to melange to support this?

Reply:
I've built melange from source, looks good. Thanks for the idea, automation'll pick it up.